### DISCUSSION TITLE
Wezterm's StartWindowDrag event and Windows Powertoys FancyZones

### DISCUSSION THREAD

**Discussion by codevogel** [wezterm/wezterm#5467] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I'm wondering if it's possible to make WezTerm's `StartWindowDrag` event can work with PowerToys FancyZones.

I show three Wezterm terminals in the gif below, each with different window decorations.
A is set to the defaults, B is set to `RESIZE`, and C is set to `INTEGRATED_BUTTONS|RESIZE`.

I can only get PowerToys FancyZones to activate when dragging the title bar in A, or the integrated title bar in C.

![tmp](https://github.com/wez/wezterm/assets/8700780/8a3e0d80-947f-462b-adc8-cf3799eeedb6)

I'm not sure if this should be a feature request for WezTerm or PowerToys. I think WezTerm makes more sense as this is a very specific case, and PowerToys is designed to only work when dragging title bars. What are your thoughts on this? Would this even be possible to do?

**Comment by GTHell** [comment]
Mine is working fine. Just a feedback!

**Reply by codevogel** [reply]
> Mine is working fine. Just a feedback!

Can you confirm that you are actually clicking in the terminal itself, and not on the titlebar?
What are your window decoration options? Does it work when you set it to just `INTEGRATED_BUTTONS|RESIZE`, and drag in the window (not in the tab bar)?
